.github/workflows/release-periodically.yml

```diff
@@ -61,8 +61,14 @@ jobs:
         if: steps.check-tag.outputs.can_bump_version
         # the repo should have an access token from the checkout action
         # https://github.com/actions/checkout#Push-a-commit-using-the-built-in-token
+        # push first the commit to ensure there isn't anything merged before starting, and then add just the new tag
         run: |
-          git push --tags origin master
+          tag_to_push=$(git describe --tags --exact-match)
+          if [ -n "${tag_to_push}" ] ; then
+            git push origin master && git push origin "$tag_to_push"
```

Comment:
This is not atomic, we need to push the commit and the tag at the same time. See: https://github.blog/2015-04-30-git-2-4-atomic-pushes-push-to-deploy-and-more/

Reply:
Thank you, that looks better, let's use it.